docs/_documentations/installoncloud.md

```diff
@@ -17,90 +17,42 @@ To setup a Codewind-ready install of Che, follow these instructions. Alternative
 
 # Table of Contents
 1. [Prerequisites](#prerequisites)
-2. [Installing Che with the operator](#installing-che-with-the-operator)
-3. [Setting up OKD and OpenShift](#setting-up-okd-and-openshift)
-4. [Setting up other Kubernetes](#setting-up-other-kubernetes)
-5. [After installing Che](#after-installing-che)
-6. [Creating the Codewind workspace](#creating-the-codewind-workspace)
-7. [Updating the version](#updating-the-version)
+2. [Installing Che with chectl](#installing-che-with-chectl)
+3. [Enable privileged and root containers to run](#enable-privileged-and-root-containers-to-run)
+4. [After installing Che](#after-installing-che)
+5. [Creating the Codewind workspace](#creating-the-codewind-workspace)
+6. [Updating the version](#updating-the-version)
 
 ## Prerequisites
 - Set up the PersistentVolume (PV) with either Network File System (NFS) or GlusterFS.
-  - For NFS, if running on IBM Cloud Private, follow these instructions:[NFS Storage Install Instructions](https://www.ibm.com/developerworks/community/blogs/fe25b4ef-ea6a-4d86-a629-6f87ccf4649e/entry/Working_with_storage?lang=en) and use 777 permissions for the folders.
+  - For NFS, we recommend 777 permissions for the exported folders and ownership of `nobody:nogroup`
```

Comment:
Add period at end of sentence.

Reply:
Instead of "recommend," can we tell the users what the benefit of the 777 permissions is? Or could we just tell them something like, "...set 777 permissions..."